loadtest/control/utils.go

```diff
@@ -13,12 +14,27 @@ import (
 	"time"
 )
 
+func init() {
+	paths := []string{"./testdata/test_text.txt", "./../../testdata/test_text.txt"}
+	var buf []byte
+	var err error
+
+	for _, p := range paths {
+		buf, err = ioutil.ReadFile(p)
+		if err != nil {
+			continue
+		}
+	}
+	words = strings.Split(string(buf), "\n")
+}
+
 const (
 	pkgPath = "github.com/mattermost/mattermost-load-test-ng/loadtest/"
 	letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
 )
 
 var re = regexp.MustCompile(`-[[:alpha:]]+`)
+var words []string
```

Comment:
This will be shared among all users right?

Reply:
Hmm, yes we can expose the function itself. Haven't thought about the actual usage :)

Reply:
Just want to make sure we don't create a race cause each user is effectively run into their own goroutine. If we can guarantee that the slice gets populated before any user gets started we should be safe I guess.

Reply:
It should: https://stackoverflow.com/a/49831018